docs/release-notes/8.9.asciidoc

```diff
@@ -0,0 +1,72 @@
+[[release-notes-header-8.9.0]]
+== 8.9.0
+
+[discrete]
+[[release-notes-8.9.0]]
+=== 8.9
+
+[discrete]
+[[known-issue-8.9.0]]
+==== Known issues
+
+* On the new Detection rule monitoring dashboard, total `Rule executions` will not always equal the sum of `Succeeded`, `Warning`, and `Failed` executions. This is expected because rules can write multiple statuses per execution. One typical example is gap detection: if a rule detects a gap in rule execution it will write an intermediate `Failed` status, then continue to run, and write a final status (such as `Warning`) before finishing its execution.
+* Rule changes can't be saved if the rule's action frequency is shorter than the rule's run interval.
+* The upload response action does not report the correct amount of available disk space. The correct amount is approximately four gigabytes. 
+
+[discrete]
+[[breaking-changes-8.9.0]]
+==== Breaking changes
+//tag::breaking-changes[]
+// NOTE: The breaking-changes tagged regions are reused in the Elastic Installation and Upgrade Guide. The pull attribute is defined within this snippet so it properly resolves in the output.
+// THIS ALSO MEANS IF YOU USE LINKS HERE, THEY SHOULD BE FULL URLS WITH NO ATTRIBUTES
+
+:pull: https://github.com/elastic/kibana/pull/
+
+There are no breaking changes in 8.9.0.
+
+//end::breaking-changes[]
+
+[discrete]
+[[deprecations-8.9.0]]
+==== Deprecations
+* Removes the option to use the legacy navigation menu ({pull}158094[#158094]).
+* Several prebuilt threat indicator match rules were deprecated and replaced with improved indicator type rules. 
+
+[discrete]
+[[features-8.9.0]]
+==== New features
+* Allows you to install the Cloud Security Posture Management (CSPM) integration via CloudFormation ({pull}159994[#159994]).
+* Creates a new dashboard, Cloud Native Vulnerability Management, that provides an overview of vulnerabilities on your cloud hosts ({pull}159699[#159699]).
+* Allows you to group vulnerabilities by resource (host) on the Vulnerabilities Findings page, and creates a Resource flyout that displays detailed vulnerability findings for individual hosts ({pull}159873[#159873], {pull}158987[#158987]).
+* Adds a new custom dashboard, "Detection rule monitoring" ({pull}159875[#159875]).
+* Allows you to anonymize event field values sent to AI Assistant ({pull}159857[#159857]).
+* Adds a *Chat* button that opens AI Assistant to the alert details flyout ({pull}159633[#159633]).
+* Updates AI Assistant to let you create and delete custom system prompts and default conversations ({pull}159365[#159365]).
+* Allows you to add alert tags ({pull}157786[#157786]).
+* Adds the ability to automatically isolate a host through a rule’s endpoint response action. 
+* Moves response actions to General Availability.
+* Adds a new response action that allows you to upload files to an endpoint that has {elastic-endpoint} installed.
```

Comment:
@kevinlog @caitlinbetz would you mind reviewing lines 46-48 when you have a moment? I couldn't find the PRs for these features so please let me know what those are if they should be included. Thank you!

Reply:
I found a couple of possible PRs to reference, but not for all three lines.